tensorflow/compiler/mlir/quantization/tensorflow/python/integration_test/quantize_model_test_base.py

```diff
@@ -44,7 +44,6 @@
 from tensorflow.python.ops import while_loop as while_loop_ops
 from tensorflow.python.ops.ragged import ragged_string_ops
 from tensorflow.python.platform import test
-from tensorflow.python.platform import tf_logging as logging
```

Comment:
This is actually needed internally. Can you add it back please? (maybe also add a comment like `  # keep` so it won't get removed later?)